lektor/admin/static/js/views/EditPage.jsx

```diff
@@ -130,6 +130,29 @@ class EditPage extends RecordEditComponent {
   }
 
   saveChanges () {
+    const path = this.getRecordPath()
+    const alt = this.getRecordAlt()
+    const newData = this.getValues()
+    utils.apiRequest('/rawrecord', { json: {
+      data: newData, path: path, alt: alt },
+    // eslint-disable-next-line indent
```

Comment:
The formatting around here looks a bit iffy. Is this suppression necessary?

Reply:
I've removed that commented line. It was in the core that way - so I just left it in there in case it served some purpose at some point - but I've removed it.